### PR TITLE
Create User button in Employee if no User ID selected

### LINKED ISSUE #7173: Automate Employee creation with User creation or vice versa
Currently users and employees need to be created separately in the system even when they are the same person. Please allow employee creation with user creation or vice versa so that the work is not duplicated. This will also negate the requirement to have duplicate fields like birth dates, phone, etc. from both screens.

### PR DESCRIPTION
**Pull request by KanchanChauhan**
Fixed https://github.com/frappe/erpnext/issues/7173

### CODE REVIEW

File: erpnext/hr/doctype/employee/employee.json [hunk 1]
```diff
@@ -2234,7 +2263,7 @@
  ], 
  "hide_heading": 0, 
  "hide_toolbar": 0, 
- "icon": "fa fa-user", 
+ "icon": "icon-user",
```
Comment (nabinhait) [inline erpnext/hr/doctype/employee/employee.json]: why icon has been overwritten? Seems like, your local sites has not migrated.

File: erpnext/hr/doctype/employee/employee.py [hunk 2]
```diff
@@ -250,3 +257,20 @@ def deactivate_sales_person(status = None, employee = None):
...
+			"field_map": {
+				"employee_name": "first_name",
+				"prefered_email": "email",
+				"gender": "gender",
+				"date_of_birth": "birth_date",
+				"cell_number": "phone",
+				"bio": "bio"
+			}
+		}}, user)
+	user.save()
```
Comment (nabinhait) [inline erpnext/hr/doctype/employee/employee.py]: user should be enabled?

Also, why use mapper for this? May be `new_doc` will be sufficient and cleaner in this case.

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/hr/doctype/employee/employee.json (hunk 1 above)
Comment (KanchanChauhan) [inline erpnext/hr/doctype/employee/employee.json]: This is some issue, every time you change something in doctype, it changes back to icon-user.

File: erpnext/hr/doctype/employee/employee.py (hunk 2 above)
Comment (KanchanChauhan) [inline erpnext/hr/doctype/employee/employee.py]: Done

**Review by nabinhait — APPROVED** [review]

**Pull request merged**